src/Microsoft.AspNet.Mvc.Core/ModelBinding/CancellationTokenModelBinder.cs

```diff
@@ -17,7 +17,13 @@ public Task<ModelBindingResult> BindModelAsync(ModelBindingContext bindingContex
             if (bindingContext.ModelType == typeof(CancellationToken))
             {
                 var model = bindingContext.OperationBindingContext.HttpContext.RequestAborted;
-                return Task.FromResult(new ModelBindingResult(model, bindingContext.ModelName, isModelSet: true));
+                var validationNode =
+                    new ModelValidationNode(bindingContext.ModelName, bindingContext.ModelMetadata, model);
```

Comment:
Thats y It doesnt has a child node, this is basically for the case when you have a property for cancellation token and there is a [Required] on the property you want to add an error. 
Note this is consistent with what was before.